src/components/sidebar-title/SidebarTitle.tsx

```diff
@@ -1,21 +1,23 @@
-import React from 'react';
+import React, { ReactNode, useState } from 'react';
 import { SSidebarTitle } from './styles/SSidebarTitle';
 import { SSidebarTitleLabel } from './styles/SSidebarTitleLabel';
 import { SSidebarTitleWrapper } from './styles/SSidebarTitleWrapper';
+import { Toggle } from '../toggle';
 
 export interface ISidebarTitleProps {
-  toggleComponent?: React.ReactNode;
-  icon?: React.ReactNode;
+  icon?: ReactNode;
+  toggle?: boolean;
 }
 
 export const SidebarTitle: React.FC<ISidebarTitleProps> = ({ ...props }) => {
+  const [value, onChange] = useState(true);
   return (
     <SSidebarTitle>
       <SSidebarTitleWrapper {...props}>
         {props.icon}
         <SSidebarTitleLabel {...props}>{props.children}</SSidebarTitleLabel>
       </SSidebarTitleWrapper>
-      {props.toggleComponent}
+      {props.toggle && <Toggle small value={value} onChange={() => onChange(!value)}></Toggle>}
```

Comment:
@steiningermona The component breaks on long words.
![Uploading Screen Shot 2021-12-01 at 12.49.15 PM.png…]()

Reply:
Also prefer destructuring and named imports for react.